### PR TITLE
chore(core): inform messaging that the processing for a message is done

### PR DESCRIPTION
**Pull request by samuelmasse**
Same thing as https://github.com/botpress/botpress/pull/5590

### CODE REVIEW

File: packages/runtime/src/runtime/messaging/messaging-service.ts [hunk 1]
```diff
@@ -120,16 +128,37 @@ export class MessagingService {
...
+      // eslint-disable-next-line @typescript-eslint/no-floating-promises
+      this.sendProcessingDone(event)
+    }
+  }
+
+  private async sendProcessingDone(event: IO.IncomingEvent) {
+    try {
+      await this.eventEngine.waitOutgoingQueueEmpty(event)
+      await this.clientsByBotId[event.botId].messages.endTurn(event.messageId!)
+      this.collectingCache.del(event.messageId!)
```
Comment (laurentlp) [inline packages/runtime/src/runtime/messaging/messaging-service.ts:156]: move into finally block

File: packages/runtime/src/runtime/messaging/messaging-service.ts [hunk 2]
```diff
@@ -120,16 +128,37 @@ export class MessagingService {
...
     event.messageId = message.id
 
     return next(undefined, true, false)
   }
 
+  public informProcessingDone(event: IO.IncomingEvent) {
+    if (this.collectingCache.get(event.messageId!)) {
+      // We don't want the waiting for the queue to be empty to freeze other messages
+      // eslint-disable-next-line @typescript-eslint/no-floating-promises
+      this.sendProcessingDone(event)
```
Comment (laurentlp) [inline packages/runtime/src/runtime/messaging/messaging-service.ts:148]: PP:

```suggestion
      void this.sendProcessingDone(event)
```

**Review by laurentlp — APPROVED** [review]

**Comment by samuelmasse** [comment]
@allardy We need to merge this and make a new release of the runtime

**Comment by allardy** [comment]
If it works i'm good with that, we'll bring back the runtime on the main repository soon anyway

**Pull request merged**